### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
Added 
1) responsive Home page

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
+}
```
Comment (ozairkhan2810) [inline .vscode/settings.json:3]: can remove this file

**Review by ozairkhan2810 — APPROVED** [review]
everything looks good, can merge

File: friends/friends.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Friends Quiz</title>
```
Comment (Kothalkarpriya) [inline friends/friends.html:8]: No CSS applied to this file?

File: friends/friends.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+</div>   
+ 
+
+<script src="/friends/friends.js"></script>
+</body>
+</html>
+
+
+
+
```
Comment (Kothalkarpriya) [inline friends/friends.html:40]: remove such extra white space, and do practise using code formatter before push

File: index.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        <a href="/rules.css"><button class="btn">Play</button></a>
+      </div>
+
+      <div class="quiz flex--cemter">
+        <img src="/Assets/Squid Game/squidGamePNG.jpeg" alt="" />
+        <div class="quiz--heading">Squid Game</div>
+        <div class="quiz--subHeading">
+          If you win you win 456 Million dollars
+        </div>
+        <a href=""><button class="btn">Play</button></a>
```
Comment (Kothalkarpriya) [inline index.html]: use # for blank href ,make it a dead link

**Review by Kothalkarpriya — CHANGES_REQUESTED** [review]
Look into my suggested changes

File: friends/friends.html (hunk 2 above)
Comment (SuyashTiwariCoder) [reply friends/friends.html:8]: done

File: friends/friends.html (hunk 3 above)
Comment (SuyashTiwariCoder) [reply friends/friends.html:40]: done

File: index.html (hunk 4 above)
Comment (SuyashTiwariCoder) [reply index.html]: done

File: .vscode/settings.json (hunk 1 above)
Comment (SuyashTiwariCoder) [reply .vscode/settings.json:3]: ok i will remove this next time

**Review by Kothalkarpriya — APPROVED** [review]
good work , you can merge

**Pull request merged**